image/system_a/data/lang/Deutsch.json

```diff
@@ -666,6 +666,7 @@
   "app_bell_relaxation_looped": "geschlungen",
   "app_bell_relaxation_loop_description": "der Titel wird abgespielt, bis Sie ihn ausschalten",
   "app_bell_relaxation_error_message": "Nicht unterstütztes Dateiformat",
+  "app_bell_relaxation_limit_error_message": "<text>File limit exceeded.<br />Not all files may display<br />correctly</text>",
```

Comment:
We have to translate this before merging.

Reply:
I know, these are placeholder, and I am waiting for translations.

Reply:
Translations are updated